charts/vastblock/templates/storage-class.yaml

```diff
@@ -52,12 +52,24 @@ metadata:
   {{- include "vastcsi.labels" $ | nindent 4 }}
 reclaimPolicy: {{ $reclaim_policy }}
 parameters:
-  subsystem: {{ $subsystem }}
-{{- range $key, $value := dict "vip_pool_name" $vip_pool_name "vip_pool_fqdn" $vip_pool_fqdn "volume_group" $volume_group "transport_type" $transport_type "fsType" $fstype "tenant_name" $tenant_name }}
-  {{- if and $value (ne $value ( quote "" )) }}
-  {{ $key }}: {{ if (kindIs "int" $value) }}{{ $value | quote }}{{ else }}{{ $value }}{{ end }}
+  subsystem: {{ $options.subsystem }}
+  {{- range $key, $value := dict 
+      "vip_pool_name" $options.vipPool 
+      "vip_pool_fqdn" $options.vipFqdn 
+      "volume_group" $options.volumeGroup 
+      "transport_type" $options.transportType 
+      "fsType" $options.fsType 
+      "tenant_name" $options.tenant }}
+    {{- if and $value (ne $value (quote "")) }}
+  {{ $key }}: {{ $value | quote }}
+    {{- end }}
+  {{- end }}
+  {{- $host_encryption := get $options "hostEncryption" }}
+  {{- if or (not $host_encryption) (eq $host_encryption "false") }}
+  host_encryption: "false"
```

Comment:
`host_encryption` here is simple boo val represented as string. Although in values.yaml it is object with many keys. If I understand correctly it is temporary solution for testing right? 
Later it can be json string via toJson func

Reply:
agreed- a json with encryption related params as keys would be nice. For now, just a way to turn the knob and test out the flow.